public/latest

```diff
@@ -0,0 +1 @@
+/Users/solomonhawk/Projects/trackomatic/public/1.1.0
```

Comment:
This symlink looks like it's not going to be portable. I think one can create a relative symlink?